.github/workflows/config/spelling_allowlist.txt

```diff
@@ -6,6 +6,7 @@ AST
 Aer
 AlmaLinux
 Anyon
+Async
```

Comment:
Pedantically speaking, "async" is not a word. Is this from example code in a comment? Can it be quoted as code?